$NetBSD: patch-ab,v 1.2 2011/09/12 06:27:21 dholland Exp $

- use export-dynamic correctly
- check for sched_get_priority_max being in -lrt

--- configure.in.start	2008-02-25 22:37:30.000000000 +0000
+++ configure.in
@@ -39,7 +39,7 @@ case "$OS" in
 		;;
 esac
 
-LDFLAGS="$LDFLAGS -export-dynamic"
+LDFLAGS="$LDFLAGS -Wl,--export-dynamic"
 
 AC_HEADER_STDC
 AC_CHECK_HEADERS([fcntl.h string.h strings.h sys/param.h unistd.h sys/un.h \
@@ -49,7 +49,10 @@ AC_CHECK_HEADERS([fcntl.h string.h strin
 		 AC_MSG_ERROR([Can't find required header files.]))
 AC_CHECK_HEADERS([sys/select.h inttypes.h limits.h stdint.h])
 
-AC_CHECK_FUNCS([sched_get_priority_max])
+AC_CHECK_FUNCS([sched_get_priority_max], ,
+	       [AC_CHECK_LIB([rt], 
+		[sched_get_priority_max], 
+		[EXTRA_LIBS="$EXTRA_LIBS -lrt"])])
 
 dnl langinfo
 AC_CHECK_HEADERS([langinfo.h])
